src/viperleed/calc/files/parameters/file_reader.py

```diff
@@ -0,0 +1,61 @@
+"""Module file_reader of viperleed.calc.classes.
```

Comment:
This does not fit with where this module is placed. It currently is in `viperleed.calc.files.parameters.file_reader`. I'm not sure now where this would make more sense. My guess is in `viperleed.calc.files`, since, while it is a class, it is specifically related to reading input files.